### DISCUSSION TITLE
Does ARFClassifier Recheck removed poor attrs?

### DISCUSSION THREAD

**Discussion by jpfeil** [online-ml/river#1482] [Q&A]
I like the idea of remove_poor_attrs, but I'm concerned that attributes that may not have performed well before may perform in the future. Does the model recheck poor attrs to see if there is signal now?

**Comment by smastelini** [comment]
Hi @jpfeil, I may need to check the code again. But as far as I recall, there is no going back once the attributes are removed. Particularly, I've never used this option extensively for the reason you mention. 

We need to keep in mind that the vanilla Hoeffding Tree algorithm (for which this feature was proposed) produces a stationary leaners.
